jenkins/pipelines/ci/tikv/tikv_ghpr_test.groovy

```diff
@@ -506,7 +506,7 @@ stage('Test') {
                                 curl -o \$i ${FILE_SERVER_URL}/download/tikv_test/${ghprbActualCommit}/\$new_string --create-dirs;
                                 chmod +x \$i;
                             done
-                            if cargo nextest run -P ci --binaries-metadata test-binaries.json --cargo-metadata test-metadata.json --partition count:${chunk_suffix}/${CHUNK_COUNT} ${EXTRA_NEXTEST_ARGS}; then
+                            if cargo nextest run -P ci --binaries-metadata test-binaries.json --cargo-metadata test-metadata.json --partition count:${chunk_suffix}/${CHUNK_COUNT} --nocapture ${EXTRA_NEXTEST_ARGS}; then
```

Comment:
Can we pass it by `EXTRA_NEXTEST_ARGS`?

Reply:
Is there any different? I suppose `EXTRA_NEXTEST_ARGS` is used for extra arguments passed from PR comments.